pylint_django/tests/test_func.py

```diff
@@ -56,6 +56,7 @@ def _file_name(test):
 
 
 TESTS = get_tests()
+TESTS.extend(get_tests('input/models'))
```

Comment:
What if you move your `models/__init__.py` changes into `tests/input/__init__.py` and adjust the location of the other files as well? So maybe we'll have something like
```
input/
        __init__.py
       authors.py (or models/authors.py)
      ... tests ...
```

I just don't want to start having multiple sub-directories if not really necessary.

Reply:
We don't need to load `'input/models'` in tests, but we need `models` package here, because it is required for FK inferring, `infer_key_classes` adds `.models` on  https://github.com/PyCQA/pylint-django/blob/87f9b4cfcaeeb212d2eb8c6889ea1f0760db9936/pylint_django/transforms/foreignkey.py#L91.